packages/itwin/tree-widget/src/tree-widget-react/components/trees/models-tree/internal/ModelsTreeVisibilityHandler.ts

```diff
@@ -54,7 +53,7 @@ import type { FilteredTree } from "./FilteredTree.js";
 
 /** @beta */
 interface GetCategoryVisibilityStatusProps {
-  categoryId: Id64String;
+  categoryId: Id64Arg;
```

Comment:
```suggestion
  categoryIds: Id64Arg;
```

Reply:
It was my suggestion to keep the same name, to avoid breaking callers of visibility handler. I just realized that visibility handler is internal, so changing the prop name would only break consumers who we already break by changing its type - the ones who use `ModelsTreeVisibilityHandlerOverrides`, likely nobody.

Let's do the rename.